templates/default/deploy-scripts/README.md

```diff
@@ -0,0 +1,112 @@
+
+At the end, you will have a Rust project setup with this boilerplate code included.
+
+# Running The Script
+
+Build and run the script from the deploy-scripts folder using
+```
+cargo run
+```
+
+The following options are necessary when running the script
+
+```
+      --node <CONCORDIUM_URL>
+          V2 API of the concordium node. [default: http://node.testnet.concordium.com:20000]
+      --account <CONCORDIUM_ACCOUNT>
+          Location path and file name of the Concordium account key file (e.g. ./myPath/4SizPU2ipqQQza9Xa6fUkQBCDjyd1vTNUNDGbBeiRGpaJQc6qX.export).
+      --modules <MODULES>
+          Location paths and names of Concordium smart contract modules. Use this flag several times \
+          if you have several smart contract modules to be deployed (e.g. --modules ./myPath/default.wasm.v1 --modules ./default2.wasm.v1).
+```
+
+The `account` parameter should be a Concordium wallet account either exported from the
+Browser wallet or the mobile wallets, or in the format emitted by the
+genesis tool.
+
+Example:
+```
+cargo run -- --node http://node.testnet.concordium.com:20000 --account ./myPath/4SizPU2ipqQQza9Xa6fUkQBCDjyd1vTNUNDGbBeiRGpaJQc6qX.export --modules ./myPath/default.wasm.v1 --modules ./default2.wasm.v1
+```
+
+# Functionalities
+
+The boilerplate code has support for the following functionalities:
+
+Read functions:
+- `estimate_energy`: To estimate the energy needed to execute one of the three write functions below.
+- `module_exists`: To check if a module has already been deployed on the chain.
+- `get_nonce`: To get the current nonce of the provided wallet account.
+
+Write functions:
+- `deploy_wasm_module`: To deploy a new smart contract module on the chain.
+- `init_contract`: To initialize a smart contract instance on the chain.
+- `update_contract`: To update a smart contract instance on the chain.
+
+Event parsing helper functions:
+- `parse_deploy_module_event`: To parse the chain events after deploying a module.
+- `parse_contract_init_event`: To parse the chain events after initialization of a smart contract instance.
+- `parse_contract_update_event`: To parse the chain events after updating a smart contract instance.
+
+The `main.rs` file has a section (marked with `// Write your own deployment/initialization script below. An example is given here.`) that you should replace with your custom logic. You can write your script using `deploy`, `initialize`, and contract `update` transactions.
+
+# Running the Example
+
+The `main.rs` file has a section (marked with `// Write your own deployment/initialization script below. An example is given here.`) that provides an example that you can run.
+
+ATTENTION: You have to have created a smart contract with the name `default` to run the given example. This can be done by answering `default` to the two questions when creating the project via `cargo-generate`.
```

Comment:
Could we use the name directly from the template instead of this message. I assume when the user reads this, they already called cargo-generate